frontends/web/src/api/bitbox02bootloader.ts

```diff
@@ -41,7 +42,7 @@ export type TVersionInfo = {
 
 export const getVersionInfo = (
   deviceID: string,
-): Promise<TStatus> => {
+): Promise<TVersionInfo> => {
```

Comment:
👍  (looks like this was not used anywhere before)

Reply:
it was me in the previous commit in this same PR :facepalm: fixed in the right commit :)